sources/packages/backend/libs/test-utils/src/index.ts

```diff
@@ -34,3 +34,4 @@ export * from "./factories/student-loan-balance";
 export * from "./factories/student-account-application";
 export * from "./factories/cra-income-verification";
 export * from "./factories/supporting-user";
+export * from "./factories/cas-supplier.factory";
```

Comment:
The other imported files don`t have the `.factory` appended. 
I would recommend following the pattern.